### ISSUE TITLE
Accessibility: The tracker image should have an alt tag

### ISSUE THREAD

**Issue by sandbergja** [subjectsplus/SubjectsPlus#1358]
SubjectsPlus uses an <img> tag to track user activity.  It should have `alt=""` so that assistive technologies know that it is not important.  Adding `aria-hidden="true"` wouldn't hurt either.

This will avoid cases where screen readers attempt to read out the whole long file name to the user.
